src/prommis/nanofiltration/diafiltration_two_salt.py

```diff
@@ -24,28 +24,30 @@
 
-#. the width of the membrane, i.e., the length of the membrane module (``membrane_width``; :math:`1 m`)
-#. the length of the membrane (``membrane_length``; :math:`100 m`)
-#. the pressure applied to the membrane system (``applied_pressure``; :math:`10 bar`)
+#. the width of the membrane, i.e., the length of the membrane module (``membrane_width``; :math:`4 \, \mathrm{m}`)
+#. the length of the membrane (``membrane_length``; :math:`40 \, \mathrm{m}`)
+#. the pressure applied to the membrane system (``applied_pressure``; :math:`15 \, \mathrm{bar}`)
 
 To run a simulation (with zero degrees of freedom) in a flowsheet, the additional following variables must be fixed to obtain zero degrees of freedom (variable names and default values are provided in parentheses):
 
-#. the volumetric flow rate of the feed (``feed_flow_volume``; :math:`100 m^3 h^{-1}`)
-#. the lithium concentration in the feed (``feed_conc_mass_lithium``; :math:`1.7 kg m^{-3}`)
-#. the cobalt concentration in the feed (``feed_conc_mass_cobalt``; :math:`17 kg m^{-3}`)
-#. the volumetric flow rate of the diafiltrate (``diafiltrate_flow_volume``; :math:`30 m^3 h^{-1}`)
-#. the lithium concentration in the diafiltrate (``diafiltrate_conc_mass_lithium``; :math:`0.1 kg m^{-3}`)
-#. the cobalt concentration in the diafiltrate (``diafiltrate_conc_mass_cobalt``; :math:`0.2 kg m^{-3}`)
+#. the volumetric flow rate of the feed (``feed_flow_volume``; :math:`100 \, \mathrm{m}^3 \, \mathrm{h}^{-1}`)
+#. the lithium concentration in the feed (``feed_conc_mol_comp[t,"Li"]``; :math:`245 \, \mathrm{mol} \, \mathrm{m}^{-3}`)
+#. the cobalt concentration in the feed (``feed_conc_mol_comp[t,"Co"]``; :math:`288 \, \mathrm{mol} \, \mathrm{m}^{-3}`)
+#. the volumetric flow rate of the diafiltrate (``diafiltrate_flow_volume``; :math:`30 \, \mathrm{m}^3 \, \mathrm{h}^{-1}`)
+#. the lithium concentration in the diafiltrate (``diafiltrate_conc_mol_comp[0,"Li"]``; :math:`14 \, \mathrm{mol} \, \mathrm{m}^{-3}`)
+#. the cobalt concentration in the diafiltrate (``diafiltrate_conc_mol_comp[0,"Co"]``; :math:`3 \, \mathrm{mol} \, \mathrm{m}^{-3}`)
 
 Model Structure
 ---------------
 
-There are three phases in the Two-Salt Diafiltration model: the retentate, the membrane, and the permeate. The retentate and the permeate are only discretized with respect to :math:`x`, while the membrane is discretized with respect to both :math:`x` and :math:`z`. The resulting system of partial differential algebraic equations is solved by discretizing with the forward finite element method.
+There are three phases in the Two-Salt Diafiltration model: the retentate, the membrane, and the permeate. The retentate and the permeate are only discretized with respect to :math:`x`, while the membrane is discretized with respect to both :math:`x` and :math:`z`. The resulting system of partial differential algebraic equations is solved by discretizing with the backward finite element method.
```

Comment:
> backward finite element method.

Pyomo DAE does not have any finite element methods implemented. Are you using the backward finite *difference* method?